src/render/PanelHeaderProvider.js

```diff
@@ -191,4 +188,10 @@ function getTemplateDocumentation(element, elementTemplates) {
   const template = getTemplate(element, elementTemplates);
 
   return template && template.documentationRef;
+}
+
+function getModelerTemplateIcon(element, iconProperty) {
+  iconProperty = iconProperty || 'zeebe:modelerTemplateIcon';
```

Comment:
Where does `iconProperty` come from? Since `PanelHeaderProvider` is generic it doesn't make sense to assume `zeebe:modelerTemplateIcon` as the default.

Reply:
@philippfromme iconProperty come from this
<img width="812" alt="image" src="https://github.com/bpmn-io/bpmn-js-properties-panel/assets/62740686/2fa3571c-1d9e-442f-bd12-939f791267b7">

Reply:
@philippfromme @nikku Have been corrected
<img width="968" alt="iShot_2024-01-04_21 23 54" src="https://github.com/bpmn-io/bpmn-js-properties-panel/assets/62740686/abc62584-8a56-4628-a53f-41596f5c2235">
<img width="1001" alt="iShot_2024-01-04_21 22 41" src="https://github.com/bpmn-io/bpmn-js-properties-panel/assets/62740686/6005ffc9-007e-4dff-9d80-6c73efa8ea1e">